SalesforceMobileSDK-iOS.podspec

```diff
@@ -16,54 +16,14 @@ Pod::Spec.new do |s|
       sdkcore.header_dir = 'Headers/SalesforceSDKCore'
-      sdkcore.prefix_header_contents = '#import <SalesforceSDKCommon/SFLogger.h>', '#import "SalesforceSDKConstants.h"'
+      sdkcore.prefix_header_contents = '#import "SFLogger.h"', '#import "SalesforceSDKConstants.h"'
       sdkcore.resource_bundles = { 'SalesforceSDKResources' => [ 'shared/resources/SalesforceSDKResources.bundle/**' ], 'Settings' => [ 'shared/resources/Settings.bundle/**' ] }
-      sdkcore.xcconfig = { 'HEADER_SEARCH_PATHS' => "${PODS_ROOT}/Headers/Public/#{s.name}/Headers", 'OTHER_CFLAGS' => '-DSQLITE_HAS_CODEC -DFMDatabase=SF_FMDatabase -DFMStatement=SF_FMStatement -DFMDatabasePool=SF_FMDatabasePool -DFMDatabaseQueue=SF_FMDatabaseQueue -DFMResultSet=SF_FMResultSet -DFMDBBlockSQLiteCallBackFunction=SF_FMDBBlockSQLiteCallBackFunction' }
```

Comment:
We need to add these flags back in, so that apps using ```FMDB``` directly don't conflict with our version of ```FMDB```.